autoPyTorch/datasets/base_dataset.py

```diff
@@ -172,8 +162,8 @@ def update_transform(self, transform: Optional[torchvision.transforms.Compose],
     def __getitem__(self, index: int, train: bool = True) -> Tuple[np.ndarray, ...]:
         """
         The base dataset uses a Subset of the data. Nevertheless, the base dataset expect
-        both validation and test data to be present in the same dataset, which motivated the
-        need to dynamically give train/test data with the __getitem__ command.
+        both validation and test data to be present in the same dataset, which is motivated
```

Comment:
the need for dynamically giving train or test data arises because the BaseDataset has both train and val data. So I think the correct way to explain would be: the base dataset expects both validation and test data to be present in the same dataset, which motivates the need to dynamically ...

Reply:
Ok, I will adapt to your comment.